test/plug_canonical_host_test.exs

```diff
@@ -5,13 +5,22 @@ defmodule PlugCanonicalHostTest do
   defmodule TestApp do
     use Plug.Router
 
-    plug(PlugCanonicalHost, canonical_host: "example.com", ignored_hosts: ["example.org"])
+    plug(PlugCanonicalHost, canonical_host: "example.com", ignore: &TestApp.ignore_proxy_requests/1)
```

Comment:
@simonprev If I use:

```elixir
plug(PlugCanonicalHost, canonical_host: "example.com", ignore: &gnore_proxy_requests/1)
```

here, I get a compilation error:

```
(CompileError) test/plug_canonical_host_test.exs:8: undefined function ignore_proxy_requests/1
```

whether the `ignore_proxy_requests` is public or private. 🤔

Reply:
I don’t know maybe because the module is in a `exs` file 🤔 mixed with the fact that the router is using macros to build the flow 🤷‍♂️